index.bs

```diff
@@ -1555,6 +1555,7 @@ partial interface MLGraphBuilder {
     1. [=Assert=]: |op| is one of "argMin", "argMax".
     1. If [=this=].{{MLGraphBuilder/[[hasBuilt]]}} is true, then [=exception/throw=] an "{{InvalidStateError}}" {{DOMException}}.
     1. If [=MLGraphBuilder/validating operand=] with [=this=] and |input| returns false, then [=exception/throw=] a {{TypeError}}.
+    1. If |input|'s [=MLOperand/shape=][|axis|] is greater than |options|.{{MLArgMinMaxOptions/outputDataType}}'s maximum value, [=exception/throw=] a {{TypeError}}.
```

Comment:
Nit: As the index starts from 0, `MAX_INT + 1` size dimension would be addressable by `int32_t` index?

Reply:
that's technically true.. but in reality, the outputDataType usually matches with the dimension's constraint, for CoreML case, the shape[axis] also won't be able to have MAX_INT32 + 1 as it's of type int32_t..

Reply:
> for CoreML case, the shape[axis] also won't be able to have MAX_INT32 + 1 as it's of type int32_t. So I think it's better to leave as <= output_type_max_value?

That makes sense. Thanks for the explanation.